src/main/java/net/sf/jabref/logic/util/strings/RtfCharMap.java

```diff
@@ -9,11 +9,11 @@ public RtfCharMap () {
         put("`i", "\\'ec");
         put("`o", "\\'f2");
         put("`u", "\\'f9");
-        put("?a", "\\'e1");
-        put("?e", "\\'e9");
-        put("?i", "\\'ed");
-        put("?o", "\\'f3");
-        put("?u", "\\'fa");
+        put("'a", "\\'e1");
```

Comment:
Yes sure, later in the RTFCharMap you would find "'a" assigned with unicode, so I thought why not just map it to the RTF value if it already exists.